modules/kummaBidAdapter.js

```diff
@@ -0,0 +1,63 @@
+var bidfactory = require('src/bidfactory.js');
+var bidmanager = require('src/bidmanager.js');
+var adloader = require('src/adloader.js');
+var utils = require('src/utils.js');
+var CONSTANTS = require('src/constants.json');
+var adaptermanager = require('src/adaptermanager');
+
+var KummaAdapter = function KummaAdapter() {
+  function _callBids(params) {
+    var bidURL;
+    var bids = params.bids || [];
+    var requestURL = window.location.protocol + '//cdn.kumma.com/pb_ortb.js?cb=' + new Date().getTime() + '&ver=1&';
+
+    for (var i = 0; i < bids.length; i++) {
+      var requestParams = {};
+      var bid = bids[i];
+
+      requestParams.pub_id = bid.params.pubId;
+      requestParams.site_id = bid.params.siteId;
+      requestParams.placement_id = bid.placementCode;
+
+      var parseSized = utils.parseSizesInput(bid.sizes);
+      var arrSize = parseSized[0].split('x');
+
+      requestParams.width = arrSize[0];
+      requestParams.height = arrSize[1];
+      requestParams.callback = '$$PREBID_GLOBAL$$._doKummaCallback';
+      requestParams.callback_uid = bid.bidId;
+      bidURL = requestURL + utils.parseQueryStringParameters(requestParams);
+
+      utils.logMessage('Kumma.prebid, Bid ID: ' + bid.bidId + ', Pub ID: ' + bid.params.pubId);
+      adloader.loadScript(bidURL);

```

Comment:
does your endpoint support `xhr` ? If it does you should use that instead of a JSONP. JSONP endpoint bidders will be deprecated in prebid 1.0.